docs/exts/logging.rst.jinja2

```diff
@@ -22,7 +22,7 @@
 
 :doc:`{{ provider }}:logging/index`
 
-{% for handler in provider_dict['handlers'] -%}
+{% for handler in provider_dict['logging'] -%}
```

Comment:
I think this is right, but just double-checking this wasn't changed by accident.

Reply:
I **think** it is right as it is, but I will check it when I rebase the final PR.

Reply:
Yep. It's ok. Double checked.